cmake/cmake_celix/BundlePackaging.cmake

```diff
@@ -195,15 +195,20 @@ function(add_celix_bundle)
     endif ()
 
     if (NOT DEFINED BUNDLE_FILENAME)
-        set(BUNDLE_FILENAME ${BUNDLE_TARGET_NAME})
-    endif ()
-
-    if (${CMAKE_BUILD_TYPE} IN_LIST CELIX_NO_POSTFIX_BUILD_TYPES)
-        set(BUNDLE_FILENAME ${BUNDLE_FILENAME}.zip)
+        set(BASE_BUNDLE_FILENAME ${BUNDLE_TARGET_NAME})
     else ()
-        set(BUNDLE_FILENAME ${BUNDLE_FILENAME}-${CMAKE_BUILD_TYPE}.zip)
+        set(BASE_BUNDLE_FILENAME ${BUNDLE_FILENAME})
```

Comment:
Why the additional BASE variable? The foreach part is conditional, so no need for different names?